invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RemoveFromCommunity/RemoveFromCommunityAction.js

```diff
@@ -60,7 +61,7 @@ export class RemoveFromCommunityAction extends Component {
     } catch (e) {
       this.setState({ error: e, loading: false });
     }
-    this.setState({ modalOpen: false });
+    !this.state.error && this.closeConfirmModal();
```

Comment:
I would suggest moving this line unchanged (without adding the conditional statement you've proposed) behind line 60 (behind the `successCallback(response.data)`) 

this will allow us avoiding the increase of complexity of the code by adding the conditional statements